### PR TITLE
Reworked theme function for light and dark mode

### PR DESCRIPTION
**Pull request by OGTor**
## 🚀 Description
Reworked themes after consultation from Yaroslav.
A toggle has been added to the settings page to switch from light to dark

## 📦 Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist (copied from README)

- [ ] Squash your changes into a single clear and thoroughly descriptive commit, split changes into multiple commits only when it contributes to readability
- [ ] Reference the GitHub issue that you are contributing on in your commit title or body
- [ ] Sign your commits, as this is required by the automated GitHub PR checks
- [ ] Ensure that the changes adhere to the project code style and formatting rules by running `npx eslint .` and `npx prettier --check ../` from the `./OwnTube.tv/` directory (without errors/warnings)
- [ ] Include links and illustrations in your pull request to make it easy to review
- [ ] Request a review by @ar9708, @OGTor, @tryklick and @mblomdahl

### CODE REVIEW

File: OwnTube.tv/src/screens/HomeScreen/styles.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from "react-native";
+
+export const styles = (theme: any) =>
```
Comment (Dykyi) [inline OwnTube.tv/src/screens/HomeScreen/styles.ts:3]: We are using TS so please forget about any in the PRs :)

**Review by Dykyi — CHANGES_REQUESTED** [review]

File: OwnTube.tv/src/screens/HomeScreen/styles.ts (hunk 1 above)
Comment (Dykyi) [reply OwnTube.tv/src/screens/HomeScreen/styles.ts:3]: Please apply it in other places as well

**Comment by Dykyi** [comment]
Nice PR!

**Review by tryklick — APPROVED** [review]

File: OwnTube.tv/src/screens/HomeScreen/styles.ts (hunk 1 above)
Comment (OGTor) [reply OwnTube.tv/src/screens/HomeScreen/styles.ts:3]: image
I get this error when trying to run it without any

i think it has something to do with `tsconfig.js` and the `"compilerOptions": {"strict": true}`

Comment (Dykyi) [reply OwnTube.tv/src/screens/HomeScreen/styles.ts:3]: This is normal, we have to keep it but fix your issue using RN types

**Review by OGTor — COMMENTED** [review]
@Dykyi is this a better approach?

**Review by Dykyi — APPROVED** [review]

**Comment by OGTor** [comment]
> Please fix your GitHub Actions pipeline so the code contribution can be reviewed over at https://ogtor.github.io/web-client/ before merging. Now it looks like this:
> image

fixed now

**Pull request merged**